src/powershell/FinOpsToolkit.psm1

```diff
@@ -2,9 +2,10 @@
 # Licensed under the MIT License.
 
 # Import the localized Data
-$script:localizedData = Import-LocalizedData -FileName 'FinOpsToolkit.strings.psd1' -BaseDirectory $PSScriptRoot
+$script:localizedData = Import-LocalizedData -FileName 'FinOpsToolkit.strings.psd1' -BaseDirectory (Join-Path -Path $PSScriptRoot -ChildPath 'en-US')
```

Comment:
to be honest, I haven't supported localized strings other than en-Us before. If we think it would be necessary then absolutely we should start tracking